src/components/_modal/pages/edition/Edition.js

```diff
@@ -309,7 +309,7 @@ export default function Wrap({
 
   return (
     <Edition
-      isLoading={manifestationIsLoading || !manifestations?.[0]}
+      isLoading={isLoading || manifestationIsLoading || !manifestations?.[0]}
```

Comment:
er den ikke lidt farlig dender sidste !manifestations?.[0] - bør det ikke tjekkes inden man overhovedet loader ?

Reply:
Virker ikke så farligt for mig, jeg har opdateret variable navn til isLoadingUserInfo, det har ikke umiddelbart noget med manifestations at gøre. 
i Edition har vi masser af conditional chaining, du må sige til. 
Du må sige til, hvis jeg overser noget

Reply:
jo men hvis der nu ikke ER en manifestation .. så vil der jo være evigt skeleton

Reply:
bedre end en crash! men ja, man kunne lave en early return og vise ingenting eller en lille sætning "vi kunne ikke finde dit materiale"